### DISCUSSION TITLE
Setting Up Redirect from '/en' to Root '/' as Default Locale in Next.js

### DISCUSSION THREAD

**Discussion by hqw567** [amannn/next-intl#636] [Q&A] [answered]
I reviewed the [example-app-router](https://github.com/amannn/next-intl/tree/main/examples/example-app-router) example, but now I need to set up a default route. For instance, in `middleware.ts`, after I have configured `defaultLocale: 'en'`, accessing `/en` should default to the `/` directory, similar to how it was used in the previous version 2. Is this possible to implement?

**Comment by amannn** [comment, marked as answer]
Yep, see [Don't use a locale prefix for the default locale](https://next-intl-docs.vercel.app/docs/routing/middleware#locale-prefix-as-needed) in the docs.

**Reply by hqw567** [reply]
Thanks.
